test/fixtures/wallet-web/.env

```diff
@@ -61,8 +61,8 @@ EDV_DATABASE_URL=mongodb://mongodb.example.com:27017
 EDV_DATABASE_PREFIX=edv_
 
 # HUB-AUTH
-AUTH_REST_IMAGE=ghcr.io/trustbloc/hub-auth
-AUTH_REST_IMAGE_TAG=0.1.7
```

Comment:
Is this supposed to be `trustbloc-cicd`? 
cc @sudeshrshetty

Reply:
yes, snapshots will be from cicd. release versions are from `trustbloc`

Reply:
yes - for snapshots we use `trustbloc-cicd`